src/adk/managers/config_manager.py

```diff
@@ -125,7 +125,6 @@ def get_application_path(self, application_name: str) -> Any:
         if application:
             if 'path' in application and os.path.exists(application['path']):
                 return application['path']
-
         return None
 
     def get_application_from_path(self, path: Path) -> Tuple[str, Dict[str, str]]:
```

Comment:
In add_application and get_application_from_path I would also suggest to change also to application_path.
You have to change the call to add_application and the code of both methods a little.